pages/tools/cli.js

```diff
@@ -0,0 +1,136 @@
+import GenericLayout from "../../components/layout/GenericLayout";
+import GithubButton from "../../components/buttons/GithubButton";
+
+import {
+  DocumentAddIcon,
+  BadgeCheckIcon,
+  GlobeIcon,
+  CodeIcon,
+} from "@heroicons/react/outline";
+
+import CodeBlock from "../../components/editor/CodeBlock";
+
+const features = [
+  {
+    name: "New files",
+    description:
+      "Use the CLI tool to quickly create new AsyncAPI files. Select from a range of templates (MMQT, WebSockets, Kafka and more...)",
+    icon: DocumentAddIcon,
+  },
+  {
+    name: "Validate",
+    description:
+      "Validate your AsyncAPI files with the CLI. Quickly get feedback to verify your AsyncAPI is within the correct format.",
```

Comment:
```suggestion
      "Validate your AsyncAPI files with the CLI. Quickly get feedback to verify your AsyncAPI file is within the correct format.",
```

Would it make sense to say there `AsyncAPI document` or `AsyncAPI file`?

Reply:
Going to add document for now 👍

Reply:
it is complicated, I personally always use `file` unless technically it doesn't make sense and then I say `document` 😄 but basically trying to avoid `document` as much as possible.